etc/httpd/global.conf

```diff
@@ -126,10 +126,9 @@ RewriteRule ^/ja/?$ http://www.mozilla.jp/ [L,R]
 RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloads/ [L,R=301]
 
 # bug 1138280
-RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://www.mozilla.jp/$1$2/notes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/((android/)?[0-9]+\.[0-9]+(\.[0-9]+)*(beta)?)/releasenotes/? http://www.mozilla.jp/$1/$2/releasenotes/ [L,R=301]
```

Comment:
My [original comment](https://github.com/mozilla/bedrock/pull/3266#discussion_r38385204) had a typo in it: my intended suggestion was to replace `\d+\.\d+(\.\d+)*` with `[0-9.]+` which is shorter, IMO more readable, and matches an arbitrary number of dotted numerals. I didn't intend to ask you to replace each `\d` with `[0-9]`-- I'm indifferent to that particular change, and I apologize for the unintended pedantry.

I see that you addressed the other changes as well, and the new version lgtm so far-- I'm going to do a bit of testing locally first and then merge if everything works as expected.